src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -62,12 +62,26 @@ public function getConfigTreeBuilder()
                     ->arrayNode('default_imagine_options')
                         ->prototype('scalar')->end()
                     ->end()
+                    ->arrayNode('mime_types')
+                        ->prototype('scalar')->end()->defaultValue(array(
+                                                                'image/jpeg',
+                                                                'image/jpg',
+                                                                'image/gif',
+                                                                'image/png',
+                                                                'image/bmp',
+                                                                'image/svg+xml',
+                                                                'image/vnd.adobe.photoshop',
+                                                                'application/pdf',
+                                                            ))
+                    ->end()
                 ->end()
             ->end()
             ->arrayNode('format_cache')
                 ->addDefaultsIfNotSet()
                 ->children()
-                    ->scalarNode('public_folder')->defaultValue(SuluMediaExtension::DEFAULT_FORMAT_CACHE_PUBLIC_FOLDER)->end()
+                    ->scalarNode('path')->defaultValue('%assetic.write_to%/uploads/media')->end()
```

Comment:
I don't really like that change, that enforces the developer to use assetic. Although we use assetic in the sulu-standard edition it's AFAIK not a total requirement until now. Could imagine that many people prefer a setup using grunt/gulp. /cc @chirimoya